activity.js

```diff
@@ -0,0 +1,114 @@
+"use strict";
```

Comment:
Yup, thanks.  (Intent: keep root to just `index.js` (and possibly `server.js` if necessary)